ui/src/modules/common/components/EmptyState.tsx

```diff
@@ -0,0 +1,87 @@
+import { Button } from "antd"
+import { PlayCircle, Plus, GitCommit } from "@phosphor-icons/react"
```

Comment:
Also please change this to 
```suggestion
import { PlayCircleIcon, PlusIcon, GitCommitIcon } from "@phosphor-icons/react"
```

as these are depracted icon, and there is a separate PR to replace them everywhere, so let's keep it consistent here too